### DISCUSSION TITLE
just curious:  why is AAC Audio "experimental" ?

### DISCUSSION THREAD

**Discussion by nemonoid** [roleoroleo/yi-hack-Allwinner-v2#793] [General]
I am using AAC Audio with the RSTP High stream (it's the only audio codec that works well with Shinobi DVR).
For the moment it has been running reliably for 7 days, and it doesn't seem to use much more CPU on the camera ( `top` still shown around 20% idle ).

Is there a particular reason why it's marked "experimental" in the web interface ?  or it's just that it has received little testing ?

**Comment by roleoroleo** [comment]
> Is there a particular reason why it's marked "experimental" in the web interface ? or it's just that it has received little testing ?

The second...
With the latest releases, it should work well.
I will remove the comment.

**Reply by nemonoid** [reply]
OK
I plan to use AAC with Shinobi longterm, 
so if I notice anything bad with the audio I can let you know.

**Reply by roleoroleo** [reply]
Ok, thanks.

**Reply by nemonoid** [reply]
Sorry, I uninstalled Shinobi (unrelated problems) and went back to Motioneye, so I cannot test AAC anymore... but it has worked reliably for the 10 days I used it.
